doc-content/enterprise-only/openshift/environment-authoring-con.adoc

```diff
@@ -14,16 +15,27 @@ If necessary, you can create separate environments, so that you can use one depl
 For {PRODUCT}, the procedures to deploy an authoring environment and a managed server environment are the same. You must first deploy an authoring environment template, consisting of {CENTRAL} and one {KIE_SERVER}. 
 
 If necessary, you can deploy additional {KIE_SERVER} templates in the same namespace to create an environment with multiple {KIE_SERVERS}. This environment can be a managed server environment for staging and production deployment of services. 
+
 endif::DM[]
+To enable {CENTRAL} to connect to additional {KIE_SERVERS}, you need to enable the `OpenShiftStartupStrategy` setting. To enable this seting, you must complete an additional manual step after deploying the authoring environment template.
```

Comment:
To enable {CENTRAL} to connect to additional {KIE_SERVERS}, enable the `OpenShiftStartupStrategy` setting. To enable this setting, you must complete an additional manual step after deploying the authoring environment template.